.github/workflows/deployment.yml

```diff
@@ -0,0 +1,51 @@
+name: Deployment
+
+on: [push, pull_request]
+
+jobs:
+  deploy-pypi:
+    name: PyPI deployment
+    runs-on: "ubuntu-latest"
+    if: github.event_name != 'push' || github.repository == 'DIRACGrid/DIRAC'
+    defaults:
+      run:
+        shell: bash -l {0}
+    steps:
+      - uses: actions/checkout@v2
+      - run: |
+          git fetch --prune --unshallow
+      - uses: conda-incubator/setup-miniconda@master
```

Comment:
Would you add a comment on what is this `conda-incubator`?

Reply:
Done. This is all about making the jobs faster (mamba saves maybe a minute or two vs conda) and the configuration easier to maintain than running the commands directly.

At some point I'll unify this with the other CI workflows so they all look similar.